roxie/ccd/ccdserver.cpp

```diff
@@ -12033,6 +12034,12 @@ class CRoxieServerIndexWriteActivity : public CRoxieServerInternalSinkActivity,
             metadata->setPropBin("_record_layout", layoutMetaSize, layoutMetaBuff);
             rtlFree(layoutMetaBuff);
         }
+        if (helper.queryOutputMeta() && helper.queryOutputMeta()->queryTypeInfo())
```

Comment:
This code (and the function it is in) COULD be commoned up. But that's arguably a separate issue.